src/main/java/com/vinylteam/vinyl/util/PropertiesReader.java

```diff
@@ -92,37 +91,15 @@ public PropertiesReader() {
         logger.info("Loaded properties in {}", getClass());
     }
 
-    public String getJdbcUser() {
-        return properties.getProperty("jdbc.user");
-    }
-
-    public String getJdbcPassword() {
-        return properties.getProperty("jdbc.password");
-    }
-
-    public String getJdbcUrl() {
-        return properties.getProperty("jdbc.url");
-    }
-
-    public String getJdbcDriver() {
-        return properties.getProperty("jdbc.driver");
-    }
-
-    public String getJdbcMaximumPoolSize() {
-        return properties.getProperty("jdbc.maximum.pool.size");
-    }
-
-    public String getAppPort() {
-        return properties.getProperty("appPort");
+    public String getProperty(String propertyName) {
+        return properties.getProperty(propertyName);
     }
 
     private void validateInputStream(InputStream inputStream) {
         if (inputStream == null) {
             RuntimeException e = new RuntimeException();
-            logger.error(".properties file not found, input stream is null", e);
+            logger.error(".properties file not found.", e);
             throw e;
         }
-        logger.debug("Input stream is valid {'inputStream':{}}", inputStream);
     }
-
```

Comment:
Why delete debug that tells that input stream is valid?

Add empty row before end of class.